### DISCUSSION TITLE
@directus/types (Item-) Types missing

### DISCUSSION THREAD

**Discussion by Dominic-Marcelino** [directus/directus#24941] [Feature Requests]
### Describe the Bug

The latest published version of the types package is missing some useful types for extensions. 

The `items.d.ts` only includes the following to basic types:

```ts
export type Item = Record<string, any>;
export type PrimaryKey = string | number;
```

https://www.npmjs.com/package/@directus/types?activeTab=code

Especially the MutationOptions type should be included too, as it's used as a parameter for the ItemService
https://github.com/directus/directus/blob/9d8f29b7926547bcee401b26dc592ffca5de5dae/api/src/types/items.ts#L24

### To Reproduce

n.A

### Directus Version

10.1.0 (directus/types)

### Hosting Strategy

Self-Hosted (Docker Image)

**Comment by rijkvanzanten** [comment]
I'm not entirely sure if this is a "bug" to leave open as an issue, or more an ongoing improvement (that is already in works!) 🤔

**Comment by Dominic-Marcelino** [comment]
> I'm not entirely sure if this is a "bug" to leave open as an issue, or more an ongoing improvement (that is already in works!) 🤔

Thought with the refactoring into the @directus/types repository this should have been included. But if not, feel free to close the issue

**Comment by rijkvanzanten** [comment]
> Thought with the refactoring into the @directus/types repository this should have been included

It will be! That refactoring effort isn't done yet, which is why I'm debating whether or not to keep this open 😄 We'll leave it open as an improvement for now 👍🏻
